src/test/java/api/support/ApiTests.java

```diff
@@ -51,6 +54,9 @@ public ApiTests() {
     sourceRecordStorageClient = ResourceClient.forSourceRecordStorage(okapiClient);
     instanceRelationshipTypeFixture = new InstanceRelationshipTypeFixture(okapiClient);
     markItemFixture = new MarkItemFixture(okapiClient);
+
+    consortiumHoldingsStorageClient = ResourceClient.forHoldingsStorage(consortiumOkapiClient);
+    consortiumItemsClient = ResourceClient.forHoldingsStorage(consortiumOkapiClient);
```

Comment:
should ResourceClient.forItemsStorage() method be called here since this client is used in the tests to get items by id?

Reply:
Yes, that's right, thank you